src/commands/expandTag.ts

```diff
@@ -0,0 +1,79 @@
+import * as vscode from 'vscode'
+import { getActiveRegularEditor } from '@zardoy/vscode-utils'
+import { findCurrentOutlineItem } from '@zardoy/vscode-utils/build/outline'
+import { equals } from 'rambda'
+import { getExtensionCommandId, getExtensionSetting, registerExtensionCommand } from 'vscode-framework'
+
+type TranslatePos = [line: number, character: number]
+const getTextSafe = (
+    document: vscode.TextDocument,
+    originalPosition: vscode.Position,
+    expandBefore: TranslatePos | undefined = [0, 0],
+    expandAfter: TranslatePos = [0, 0],
+) => {
+    const positionTranslateSafe = (pos: vscode.Position, translate: TranslatePos) => {
+        const newTranslate = [...translate]
+        if (pos.line + translate[0] < 0) newTranslate[0] = 0
+        if (pos.character + translate[1] < 0) newTranslate[1] = 0
+        return pos.translate(...newTranslate)
+    }
+
+    const positionBefore = positionTranslateSafe(originalPosition, expandBefore)
+    const positionAfter = positionTranslateSafe(originalPosition, expandAfter)
+    return document.getText(new vscode.Range(positionBefore, positionAfter))
+}
+
+const outlineBasedLangs = ['vue', 'svelte']
```

Comment:
If you are copy-pasting it to vue-based ext, you should delete svelete, you even should inline this var